bindings/electron/README.md

```diff
@@ -1,9 +1,41 @@
 # libparsec-electron-bindings
 
+## Build
+
+tl;dr: `../../make.py er`
+
+```shell
+# First run (to install dependencies)
+../../make.py electron-dev-install
+# or just
+../../make.py ei
+# Subsequent runs
+../../make.py electron-dev-rebuild
+# Or just
+../../make.py er
+```
```

Comment:
I'll split the commands to be able to copy them independently with the "copy" button

````suggestion
First run (to install dependencies)
```shell
../../make.py ei    # ei is an alias for electron-dev-install
```

Subsequent runs
```shell
../../make.py er    # er is an alias for electron-dev-rebuild
```
````